src/Reflection/Php/PhpMethodFromParserNodeReflection.php

```diff
@@ -69,7 +69,13 @@ public function isPublic(): bool
 
 	public function getReturnType(): Type
 	{
-		if ($this->getName() === '__construct') {
+		static $methodsAlwaysWithVoid = [
+			'__construct' => null,
+			'__destruct' => null,
+			'__clone' => null,
+		];
```

Comment:
What about DRYing the changes into one place? public constant maybe?

Reply:
@ondrejmirtes Where should I put it? :)